.circleci/config.yml

```diff
@@ -131,6 +135,12 @@ jobs:
       - install_dependencies
       - restore_workspace
       - bundle_test
+  bundle_size_test:
```

Comment:
No new job please, just keep it simple: https://github.com/commercetools/ui-kit/pull/914#discussion_r300316270